.github/dependabot.yml

```
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

```suggestion
      interval: "weekly"
    allow:
      # Allow both direct and indirect updates for all packages.
      - dependency-type: "all"
```

As in the CLI repository, allowing Dependabot to update 'indirect' dependencies too.